schematic/manifest/generator.py

```diff
@@ -178,27 +175,8 @@ def callback(request_id, response, exception):
         batch.execute()
 
 
-    def _get_valid_values_from_jsonschema_property(self, prop:dict) -> List[str]: 
```

Comment:
Note: this function needs to stay in develop; similarly there is logic below that needs to be preserved. 

@sujaypatil96 any chance you can apply your changes over the latest code in develop?